addons/mail/static/src/models/thread/thread.js

```diff
@@ -1598,6 +1582,16 @@ function factory(dependencies) {
             compute: '_computeNeedactionMessages',
             dependencies: ['messages'],
         }),
+        /**
+         * Not a real field, used to trigger `_onIsServerPinnedChanged` when one of
+         * the dependencies changes.
+         */
+        onIsServerPinnedChanged: attr({
+            compute: '_onIsServerPinnedChanged',
+            dependencies: [
+                'isServerPinned',
+            ],
+        }),
```

Comment:
Comment's poor. Only the "Not a real field" is relevant...
Give more meaningful info of use of this field.

Reply:
I just based my comments on the ones written for fold state ...

Reply:
Yes, I think this is fine honestly. The current comment explains perfectly why the field exists and what it should be used for (= to call a method on dependency change, nothing else). The rest should be documented on the method.